plugins/modules/zos_mvs_raw.py

```diff
@@ -3090,6 +3097,49 @@ def get_content(formatted_name, binary=False, from_encoding=None, to_encoding=No
         return stdout
 
 
+def modify_contents(contents):
```

Comment:
Something in the workflow of this method is weird, so if we have a list, we add the char '\n' in each line and then pass it to `add_space` function but if is not a list, we split by the '\n' and then convert it into a list (this ends up being the same case as above right?) and then pass it to `add_space`, so you can either pass an str or a list into `add_space` but the conversion to the other type is performed anyways? 

I think that can be reduced to something like this?
```
if not isinstance(contents, list):
    if contents[0] != " " or contents[0] != "-":
        contents = "  {0}".format(contents)
    contents = list(contents.split("\n"))
contents = add_space(contents)
contents = "\n".join(contents)
return contents
```

Reply:
I now should work on that way but is required that logic, not sure it helps now.

Reply:
I didn't get it. could you explain?

Reply:
Never mind thanks for the idea

Reply:
done